code/modules/antagonists/heretic/magic/mansus_grasp.dm

```diff
@@ -90,6 +90,7 @@
 	hit.adjustBruteLoss(10)
 	if(iscarbon(hit))
 		var/mob/living/carbon/carbon_hit = hit
+		carbon_hit.cultslurring += 6
```

Comment:
Are you sure this is 6 seconds and not 12 seconds? it seems like it's reduced by 1 per life tick
12 seconds would be far to long and truthfully 6 seconds might be as well too - maybe 4 seconds (+= 2)?

Reply:
So change it to +=2 and it's good to go, right? Apologies for taking so long on this.

Reply:
Sorry, didn't notice - +=2 is good